clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/logging/JGroupsLogger.java

```diff
@@ -155,6 +154,6 @@
     IllegalArgumentException failedToResolveSocketBinding(@Cause UnknownHostException cause, OutboundSocketBinding binding);
 
     @LogMessage(level = WARN)
-    @Message(id = 29, value = "Ignoring unrecognized %s properties: %s")
-    void ignoredProperties(String protocol, Map<String, String> properties);
+    @Message(id = 29, value = "Protocol %s is obsolete and will be auto-updated to %s")
+    void legacyProtocol(String legacyProtocol, String targetProtocol);
```

Comment:
It looks like this message has been in a release, so it should probably be commented out and a new ID assigned to the new message.

Reply:
Fixed.